src/main/scala/io/scalac/tezos/translator/routes/utils/ReCaptcha.scala

```diff
@@ -74,23 +74,33 @@ object ReCaptcha {
           Left(Error("Can't do request to verify captcha"))
       }
   }
-
+  protected def checkScore(score:Option[Float]): Either[ErrorResponse, Unit] = {
+    score match {
+      case Some(number) =>
+       if (number > 0.35)
+         ().asRight
+       else
+         (Error ("You are bot"), StatusCode.Unauthorized).asLeft
+      case _ => (Error (s"""Google response was succes but doesn't have "Score", Probably you don't used reCAPTCHA V3"""), StatusCode.Unauthorized).asLeft
```

Comment:
`Google response was successful but "score" field is empty. You probably did not use reCAPTCHA V3`